### PR TITLE
Prefer 'compose.yaml' as per compose spec

### PR DESCRIPTION
**Pull request by helge000**
As per [compose spec](https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file), we should at least support, or better yet prefer `compose.yaml` (as implemented here):

>The default path for a Compose file is compose.yaml (preferred) or compose.yml in working directory

### CODE REVIEW

File: podman_compose.py [hunk 1]
```diff
@@ -846,9 +850,9 @@ def _parse_compose_file(self):
                 "container-compose.override.yml",
                 "container-compose.override.yaml"
             ]))
-        files = args.file
+        files = args.file[0]
```
Comment (helge000) [inline podman_compose.py:853]: I guess you would rather have me remove this; however I strongly believe this is the right thing to do as it could lead to unexpected results. That said, this discussion would be best for another PR, then :)

Comment (muayyad-alsadi) [reply podman_compose.py:853]: yes, please remove this.
